pkgs/development/python-modules/playwright/default.nix

```diff
@@ -52,13 +52,12 @@ buildPythonPackage rec {
     git commit -m "workaround setuptools-scm"
 
     substituteInPlace pyproject.toml \
```

Comment:
for the next PRs (it has limitations too), but I think there is `pythonRelaxDeps` that could help with that

Reply:
pythonRelaxDeps already used below, this is replacement of the build dependenices https://github.com/microsoft/playwright-python/blob/ca0cf3efdaaa5f9eb3b49a0331eb84b1908620ac/pyproject.toml#L2